client/my-sites/stats/site.jsx

```diff
@@ -613,6 +623,15 @@ export default connect(
 		const upsellModalView =
 			config.isEnabled( 'stats/paid-wpcom-v2' ) && getUpsellModalView( state, siteId );
 
+		const siteCreatedTimeStamp = getSiteOptions( state, siteId ?? 0 )?.created_at;
+
+		// TODO: update the date to the release date when the feature is ready.
+		const redirectToPurchase =
+			config.isEnabled( 'stats/checkout-flows-v2' ) &&
+			isJetpack &&
+			siteCreatedTimeStamp &&
+			new Date( siteCreatedTimeStamp ) > new Date( '2024-01-01' );
```

Comment:
Is this compared date `2024-01-01` a temporary one?

Reply:
Yes, we will change it to the release date just before the release. I used this date so people can test it on the live branches and don't have to checkout branches for new pull requests.